packages/manager/apps/hycu/src/utils/iam.constants.ts

```diff
@@ -1,4 +1,5 @@
 export const IAM_ACTIONS = {
+  licenseHycuApiOvhGet: 'licenseHycu:apiovh:license/get',
```

Comment:
Can `get` be finally managed with IAM ?

Reply:
It's the rule for download license. I thinked It's relevant for this case